pythoncalculator/mutliply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
returning an addition not a multiplication 
```suggestion
    return x * y
```